samples/grids/data-grid/cell-editing/wwwroot/index.html

```diff
@@ -28,8 +27,10 @@
     </div>
     <!-- importing blazor bundle for IG controls: -->
     <script src="_content/IgniteUI.Blazor/app.bundle.js"></script>
+    <script src="TemplateColumnScript.js"></script>
```

Comment:
please rename this file to "DataGridCellEditing.js"

Reply:
see commit 2